acceptance_tests/features/add_a_survey.feature

```diff
@@ -10,12 +10,12 @@ Feature: Add a survey
 
 
   @fixture.setup.data.with.enrolled.respondent.and.additional.iac
+  @skip
```

Comment:
Have spoken to @ryangrundy7 and confirmed that this test is left in place to avoid issues with VCS down the line.

Added tech debt card to address this later: https://trello.com/c/R7M5uhMM